lib/animas/animasSimulator.js

```diff
@@ -74,6 +74,21 @@ exports.make = function(config){
     return e;
   }
 
+  function validateDuration(basal) {
+    var fiveDays = (5 * 1440 * sundial.MIN_TO_MSEC);
+    if(basal.isAssigned('duration')) {
+      if(basal.duration > fiveDays) {
+        //flat-rate basal
+        basal.duration = fiveDays;
+        annotate.annotateEvent(basal, 'animas/basal/flat-rate');
```

Comment:
Do you feel like this is the right annotation even for the suspends? Feels different to me... a long-running suspend b/c of a pump vacation isn't really the same thing as a scheduled single rate, other than having the same data limitations.